nixos/tests/hydra/default.nix

```diff
@@ -7,9 +7,8 @@ let
      { trivial = builtins.derivation {
          name = "trivial";
          system = "x86_64-linux";
-         PATH = coreutils;
-         builder = shell;
-         args = ["-c" "touch $out; exit 0"];
+         builder = "/bin/sh";
```

Comment:
Regarding what you said in the commit message, I think `/bin/sh` comes from the busybox package which is bind mounted at the build environment creation (and not from the VM `/bin/sh`).